pkg/icingadb/db.go

```diff
@@ -542,7 +542,7 @@ func (db *DB) CreateIgnoreStreamed(
 	ctx context.Context, entities <-chan database.Entity, onSuccess ...OnSuccess[database.Entity],
 ) error {
 	first, forward, err := com.CopyFirst(ctx, entities)
-	if first == nil {
+	if err != nil {
 		return errors.Wrap(err, "can't copy first entity")
 	}
```

Comment:
Changing the error handling in this way sounds like a good idea (given that the function returns `error` already), but then the following case within the function should probably be changed to also return an error in this case:

https://github.com/Icinga/icingadb/blob/5d182aad64891e7007b7d90320bccce0c933d437/pkg/com/com.go#L52-L54

Reply:
True, fixed.